lib/inspec_tools/plugin_cli.rb

```diff
@@ -36,8 +36,9 @@ class CliCommand < Inspec.plugin(2, :cli_command) # rubocop:disable Metrics/Clas
       option :separate_files, required: false, type: :boolean, default: true, aliases: '-s'
       option :replace_tags, type: :array, required: false, aliases: '-r'
       option :metadata, required: false, aliases: '-m'
+      option :control_id, required: false, enum: %w{ruleID vulnID}, aliases: '-c', default: 'vulnID'
       def xccdf2inspec
-        xccdf = InspecTools::XCCDF.new(File.read(options[:xccdf]), options[:replace_tags])
+        xccdf = InspecTools::XCCDF.new(File.read(options[:xccdf]), options[:control_id] == 'vulnID',options[:replace_tags])
```

Comment:
I guess we're not running rubocop on this project, anyway there is a space missing between the comma and options here.